truffle/com.oracle.truffle.api.debug/src/com/oracle/truffle/api/debug/Debugger.java

```diff
@@ -234,6 +235,29 @@ public Breakpoint setTagBreakpoint(int ignoreCount, SyntaxTag tag, boolean oneSh
     }
 
     /**
+     * Check whether the debugger is associated with an ongoing execution.
+     *
+     * @return <code>true</code> if there is an ongoing execution.
+     * @since 0.13
+     */
+    public boolean isExecuting() {
+        return executing;
+    }
+
+    /**
+     * Request a pause.
+     * As soon as the execution arrives at a node holding a debugger tag,
+     * {@link SuspendedEvent} is emitted.
+     * @since 0.13
+     */
+    public void pause() {
```

Comment:
Description of a threading behavior of this method (which is different than for other methods) is necessary.